### PR TITLE
Fix logpull_options validation curl in api-configuration.md

### PR DESCRIPTION
**Pull request by swoodford-clear**
* Replace invalid end quote
* Add necessary auth headers
* Format nicely with indentation for readability

### CODE REVIEW

**Comment by swoodford-clear** [comment]
Unsure why the "spell-check" is failing, my changes had nothing to do with that line and use of the word special is warranted.

File: content/logs/get-started/api-configuration.md [hunk 1]
```diff
@@ -172,10 +172,13 @@ The **CVE-2021-44228** parameter can only be set through the API at this time. U
 To check if the selected **logpull_options** are valid:
 
 ```bash
-$ curl -s -XPOST https://api.cloudflare.com/client/v4/zones/{zone_identifier}/logpush/validate/origin 
--d '{ "logpull_options":"fields=RayID,ClientIP,EdgeStartTimestamp&timestamps=rfc3339&CVE-2021-44228=true”,
-"dataset": "http_requests", 
-}' | jq .
+$ curl -s -X POST https://api.cloudflare.com/client/v4/zones/{zone_identifier}/logpush/validate/origin \
```
Comment (angelampcosta) [inline content/logs/get-started/api-configuration.md]:
```suggestion
curl -s -X POST https://api.cloudflare.com/client/v4/zones/{zone_identifier}/logpush/validate/origin \
```

File: content/logs/get-started/api-configuration.md [hunk 2]
```diff
@@ -172,10 +172,13 @@ The **CVE-2021-44228** parameter can only be set through the API at this time. U
...
+     -H "X-Auth-Email: <EMAIL>" \
```
Comment (angelampcosta) [inline content/logs/get-started/api-configuration.md]:
```suggestion
--header "X-Auth-Email: <EMAIL>" \
```

File: content/logs/get-started/api-configuration.md [hunk 3]
```diff
@@ -172,10 +172,13 @@ The **CVE-2021-44228** parameter can only be set through the API at this time. U
...
+     -H "X-Auth-Key: <API_KEY>" \
```
Comment (angelampcosta) [inline content/logs/get-started/api-configuration.md]:
```suggestion
--header "X-Auth-Key: <API_KEY>" \
```

File: content/logs/get-started/api-configuration.md [hunk 4]
```diff
@@ -172,10 +172,13 @@ The **CVE-2021-44228** parameter can only be set through the API at this time. U
...
+     -d '{
```
Comment (angelampcosta) [inline content/logs/get-started/api-configuration.md]:
```suggestion
 --data '{
```

File: content/logs/get-started/api-configuration.md [hunk 5]
```diff
@@ -172,10 +172,13 @@ The **CVE-2021-44228** parameter can only be set through the API at this time. U
...
 
 ```bash
-$ curl -s -XPOST https://api.cloudflare.com/client/v4/zones/{zone_identifier}/logpush/validate/origin 
--d '{ "logpull_options":"fields=RayID,ClientIP,EdgeStartTimestamp&timestamps=rfc3339&CVE-2021-44228=true”,
-"dataset": "http_requests", 
-}' | jq .
+curl -s -X POST https://api.cloudflare.com/client/v4/zones/{zone_identifier}/logpush/validate/origin \
+--header "X-Auth-Email: <EMAIL>" \
+--header "X-Auth-Key: <API_KEY>" \
+ --data '{
```
Comment (angelampcosta) [inline content/logs/get-started/api-configuration.md]:
```suggestion
--data '{
```

File: content/logs/get-started/api-configuration.md [hunk 6]
```diff
@@ -172,10 +172,13 @@ The **CVE-2021-44228** parameter can only be set through the API at this time. U
...
+     "logpull_options": "fields=RayID,ClientIP,EdgeStartTimestamp&timestamps=rfc3339&CVE-2021-44228=true",
```
Comment (angelampcosta) [inline content/logs/get-started/api-configuration.md]:
```suggestion
  "logpull_options": "fields=RayID,ClientIP,EdgeStartTimestamp&timestamps=rfc3339&CVE-2021-44228=true",
```

File: content/logs/get-started/api-configuration.md [hunk 7]
```diff
@@ -172,10 +172,13 @@ The **CVE-2021-44228** parameter can only be set through the API at this time. U
...
+     "dataset": "http_requests"
```
Comment (angelampcosta) [inline content/logs/get-started/api-configuration.md]:
```suggestion
  "dataset": "http_requests"
```

File: content/logs/get-started/api-configuration.md [hunk 8]
```diff
@@ -172,10 +172,13 @@ The **CVE-2021-44228** parameter can only be set through the API at this time. U
...
+     }' | jq .
```
Comment (angelampcosta) [inline content/logs/get-started/api-configuration.md]:
```suggestion
}' | jq .
```

File: content/logs/get-started/api-configuration.md [hunk 9]
```diff
@@ -172,9 +172,12 @@ The **CVE-2021-44228** parameter can only be set through the API at this time. U
...
-$ curl -s -XPOST https://api.cloudflare.com/client/v4/zones/{zone_identifier}/logpush/validate/origin 
--d '{ "logpull_options":"fields=RayID,ClientIP,EdgeStartTimestamp&timestamps=rfc3339&CVE-2021-44228=true”,
-"dataset": "http_requests", 
+curl -s -X POST https://api.cloudflare.com/client/v4/zones/{zone_identifier}/logpush/validate/origin \
+--header "X-Auth-Email: <EMAIL>" \
+--header "X-Auth-Key: <API_KEY>" \
+--data '{
+  "logpull_options": "fields=RayID,ClientIP,EdgeStartTimestamp&timestamps=rfc3339&CVE-2021-44228=true",
+  "dataset": "http_requests"
 }' | jq .
```
Comment (pedrosousa) [inline content/logs/get-started/api-configuration.md]:
```suggestion
curl https://api.cloudflare.com/client/v4/zones/{zone_identifier}/logpush/validate/origin \
--header "X-Auth-Email: <EMAIL>" \
--header "X-Auth-Key: <API_KEY>" \
--data '{
  "logpull_options": "fields=RayID,ClientIP,EdgeStartTimestamp&timestamps=rfc3339&CVE-2021-44228=true",
  "dataset": "http_requests"
}'
```
Suggestions according to our [curl guidelines](https://developers.cloudflare.com/style-guide/api-content-strategy/guidelines-for-curl-commands/).

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**